pandas/core/reshape/concat.py

```diff
@@ -262,7 +262,13 @@ def __init__(self, objs, axis=0, join='outer', join_axes=None,
         ndims = set()
         for obj in objs:
             if not isinstance(obj, NDFrame):
-                raise TypeError("cannot concatenate a non-NDFrame object")
+
+                # multiline string concat is safer with whitespace for regex
+                # at test_concat.py:181
+                msg = """cannot concatenate object of type "{0}";"""
+                msg += " only pd.Series, pd.DataFrame, and pd.Panel"
```

Comment:
no need to mention ``pd.Panel`` as they are deprecated

Reply:
Ahh, were there plans to maintain it in some "LTS" pre 1.0 release, or is it full-on deprecated?

Reply:
are you offering to do this ???
haven’t really decided what to do with this yet

Reply:
No, that sounds decidedly outside the scope of the time I have right now. While I was aware of the deprecation, I thought there was discussion about it being passively maintained. I don't have any horses in the pd.Panel race. Just curious.